src/BenchmarkDotNet.Analyzers/BenchmarkDotNet.Analyzers.csproj

```diff
@@ -9,7 +9,7 @@
     <NoWarn>$(NoWarn);CS1591</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.14.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="5.0.0-2.final" />

```

Comment:
We might actually need to downgrade the version rather than upgrade. https://github.com/dotnet/roslyn/discussions/81256 Package versions should be forwards compatible. Maybe it's an issue with the brand new sdk?